### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by pnbr0014** [ErikBongers/streamdeck-lightroom#11]
Hey trying to figure out if this still works at all. there are no files that can be opened with lightroom and lr.exe does not do anything

Bruce.

**Comment by ErikBongers** [comment]
No, it's too old and I'm not planning to upgrade it. Sorry. On 16 Apr 2024 06:54, pnbr0014 ***@***.***> wrote:
Hey trying to figure out if this still works at all. there are no files that can be opened with lightroom and lr.exe does not do anything
Bruce.

**Comment by pnbr0014** [comment]
aw man thats a shame. only option seems to be to pay 65 bucks for LR superkeys

**Comment by pnbr0014** [comment]
plugin still works. and supports more commands than listed too. I know you aren't supporting it but if you were feeling a little happy one day would be nice to add a hold to repeat command function so that I could hold a button and it would continue to increment or decrement
